casepro/backend/tests/test_rapidpro.py

```diff
@@ -868,19 +868,32 @@ def test_push_outgoing(self, mock_create_broadcast, mock_send_raw_email):
 
     @patch("dash.orgs.models.TembaClient.get_contacts")
     @patch("dash.orgs.models.TembaClient.create_contact")
-    def test_push_contact(self, mock_get_contacts, mock_create_contact):
+    def test_push_contact(self, mock_create_contact, mock_get_contacts):
         """
         If a contact is added in CasePro,
         it should be added in RapidPro and the uuid should match
         """
         # Contact does not exist, so push contacts is called
```

Comment:
I think this comment is confusing, maybe it should be `# contact does exist in RapidPro`?